app/views/hyrax/base/_attribute_rows.html.erb

```diff
@@ -11,7 +11,7 @@
 <%= presenter.attribute_to_html(:institution, html_dl: true) %>
 <%= presenter.attribute_to_html(:org_unit, label: "Organisational unit", html_dl: true) %>
 <%= presenter.attribute_to_html(:project_name, html_dl: true) %>
-<%= render "hyku_addons/funder/show", presenter: presenter %>
+<%= render "hyku_addons/funder/show", presenter: presenter if presenter.respond_to?(:funder) %>
```

Comment:
I think it may be good to eventually get these to use the `presenter.attribute_to_html` pattern that all of the other fields use.